code/src/components/AddTask.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+
+import uniqid from 'uniqid'
+import tasks from 'reducers/tasks'
+import TaskList from 'components/TaskList'
+import CompletedTasks from 'components/CompletedTasks'
+
+const MainContainer = styled.div`
+display: flex;
+flex-direction: column;
+align-items: center;
+`;
+
+const TodoContainer = styled.div`
+	display: flex;
+	align-items: flex-start;
+	flex-direction: column;
+	background-color: rgb(255, 238, 238);
+    min-width: 15rem;
+    width: 20vw;
+	margin: 0.5rem auto;
+    padding: 2rem;
+    border-radius: 15%;
+`;
+
+const Title = styled.h1`
+    display: flex;
+    align-items: center;
+	font-size: 1.5rem;
+    padding: 2.5rem;
+    color: rgb(255, 238, 238);
+`;
+
+const AddButton = styled.button`
+    padding: 0.6rem;
+    color: rgb(255, 238, 238);
+    background-color: rgb(230, 9, 101);
+    cursor: pointer;
+    border-radius: 75%;
+    font-weight: 700;
+    margin-left: 0.5rem;
+    
+    &:hover {
```

Comment:
If you want you could make it even more clear that the button is disabled after you've written a certain amount of text by using &:disabled and making the buttons background-color with lower opacity for example. I can see that you have taken away the border for the button when the user has written 25 letters but maybe it can be even more clear with some change of the background or so :D